Makefile

```diff
@@ -26,6 +26,18 @@ build:
 run:
 	RECTIME=30 operator-sdk up local --namespace=""
 
+.PHONY: setup/service_account
+setup/service_account:
```

Comment:
Should we have a make target to remove the service account as well ? Did have issues with this in the passed with integreatly's service account locking me out. 

Wait its tied to the namespace, so as soon as the namespace is gone it should be gone right ?

Reply:
Yeah it is all added to the `namespace`, just have to remember to log yourself back in after running this target and everything should be good